proto/api/v2/compliance_results_service.proto

```diff
@@ -95,27 +99,42 @@ message ListComplianceScanResultsResponse {
 
 service ComplianceResultsService {
     // GetComplianceScanResultsOverview lists current scan configurations with most recent results overview that match the query
+    // Supported optional RawQuery query fields which can be combined
+    // - scan: id(s) of the compliance scan
+    // - cluster: id(s) of the cluster
+    // - profile: id(s) of the profile
```

Comment:
Are you suggesting that if there are other fields in the query an error will be returned?

Reply:
That's what my words say, but was not I meant.  I was trying to help the UI team understand the API as to what they could search on.  I will come up with some words to soften this.